### PR TITLE
Replaced Bootstrap to Intro Js in Vote Activty

### PR DESCRIPTION
**Pull request by NischayGoyal1**
### @llaske Please Review it

Screenshot (45)

### CODE REVIEW

File: activities/Vote.activity/js/activity.js [hunk 1]
```diff
@@ -640,168 +640,168 @@ var app = new Vue({
 			if(this.settings) {
 				steps = steps.concat([
 					{
-						element: ".poll-card#0",
-						placement: "right",
+						element: ".poll-card [id='0']",
```
Comment (disha1202) [inline activities/Vote.activity/js/activity.js:643]:
```suggestion
						element: ".poll-card[id='0']",
```

**Comment by NischayGoyal1** [comment]
Thanks @disha1202

**Comment by llaske** [comment]
I guess you're talking about Vote activity, not Write :laughing: 

There is an issue on the popup when the activity is shared: scrollbars appear.

image

**Comment by NischayGoyal1** [comment]
I have fixed this issue @llaske , along with removing some unwanted files
Please Review it :)

**Comment by llaske** [comment]
Good job @NischayGoyal1 . Thanks.

**Pull request merged**